lib/test/t_IntegrationExpansion_std.cxx

```diff
@@ -0,0 +1,154 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The test file of the IntegrationExpansion class
+ *
+ *  Copyright 2005-2022 Airbus-EDF-IMACS-ONERA-Phimeca
```

Comment:
```suggestion
 *  Copyright 2005-2023 Airbus-EDF-IMACS-ONERA-Phimeca
```

Reply:
I fixed all the date issues in one shot (including the one in OTIterativeStat.hxx)